en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
-gui.xaero_delete_world_msg2=This action cannot be reverted. Make sure you know what you're doing.
-gui.xaero_delete_multi_world=Delete Sub-world
-gui.xaero_delete_multi_world_msg1=Would you like to delete all waypoints data for the selected sub-world?
-gui.xaero_delete_multi_world_msg2=This action cannot be reverted. Make sure you know what you're doing.
-gui.xaero_always_show_distance=Always Disp. WP Dist.
+gui.xaero_delete_world_msg1=Would you like to delete all waypoint data from the selected world/server?
+gui.xaero_delete_world_msg2=This action cannot be undone! Make sure you know what you are doing.
+gui.xaero_delete_multi_world=Delete Sub-World
+gui.xaero_delete_multi_world_msg1=Would you like to delete all waypoint data from the selected sub-world?
+gui.xaero_delete_multi_world_msg2=This action cannot be undone! Make sure you know what you are doing.
+gui.xaero_always_show_distance=Always Disp. Dist. to WP
 gui.xaero_item_tooltip=Item Tooltip
 gui.xaero_entity_tooltip_settings=Item Tooltip Settings
 gui.xaero_item_tooltip_min_lines=Minimum Lines
-gui.xaero_item_tooltip_time=Show For
-gui.xaero_player_names=Player Head Names
+gui.xaero_item_tooltip_time=Display For
+gui.xaero_player_names=Player Names Under Heads
 gui.xaero_multiply_all_by_8=Sub-World Coordinates * 8
 gui.xaero_divide_all_by_8=Sub-World Coordinates / 8
-gui.xaero_multiply_msg1=This option should only be used to fix incorrect waypoint coordinates.
-gui.xaero_multiply_msg2=Are you sure you would like to MULTIPLY all sub-world coordinates by 8?
-gui.xaero_divide_msg2=Are you sure you would like to DIVIDE all sub-world coordinates by 8?
+gui.xaero_multiply_msg1=This option should be used only to fix incorrect waypoint coordinates.
+gui.xaero_multiply_msg2=Are you sure you would like to multiply all sub-world coordinates by 8?
+gui.xaero_divide_msg2=Are you sure you would like to divide all sub-world coordinates by 8?
 gui.xaero_share=Share
-gui.xaero_share_msg1=Are you sure you would like to share this waypoint with §cEVERYONE§f in chat?
-gui.xaero_share_msg2=Please make sure you don't expose a secret location.
-gui.xaero_roof=Roof
+gui.xaero_share_msg1=Are you sure you would like to share this waypoint with §cEVERYONE§f in the chat?
+gui.xaero_share_msg2=Make sure you do not expose a secret location!
+gui.xaero_roof=Ceiling
 gui.xaero_partial=Partial
 gui.xaero_full=Full
-gui.xaero_show_light_level=Show Light Level
+gui.xaero_show_light_level=Display Light Level
 gui.xaero_up=[Go Up]
 gui.xaero_down=[Go Down]
 gui.xaero_display_all_sets=Toggle All WP Sets Render
+gui.xaero_waypoint_name=waypoint name
 gui.xaero_render_layer=Overlay Render Layer
 gui.xaero_show_time=Display Game Time
 gui.xaero_24h=24h
 gui.xaero_12h=12h
 gui.xaero_temporary=temporary
 gui.xaero_restore=Restore
-gui.xaero_biomes_vanilla=Biomes In Vanilla Mode
-gui.xaero_potion_effects_blink=Blink When Low
+gui.xaero_biomes_vanilla=Biomes in Vanilla Color Mode
```

Comment:
Now since version 20.24.0 this button label barely fits and doesn't simulate having left and right padding, so I think it's better to shorten it somehow.

Reply:
There isn't supposed to be left and right padding though. It just happens to look that way for shorter labels.

Reply:
IMO button labels with (simulated) padding always look a little better. But if you like the current state, I'll leave it as is. Simply let me know.

Reply:
I know that padding looks better but I think that conveying the meaning is more important here.